### ISSUE TITLE
Mod Suggestion: 

### ISSUE THREAD

**Issue by james-pre** [xJon/The-1.12.2-Pack#971]
### Was this feature suggested before?

- [X] No, it was not.

### What is your suggestion?

Please add:

[Metallurgy 4: Reforged](https://www.curseforge.com/minecraft/mc-mods/metallurgy-4-reforged): Adds lots of material options for gear, and I've found it is commonly included in other packs.

[Draconic Evolution](https://www.curseforge.com/minecraft/mc-mods/draconic-evolution): Adds end-game gear, though I haven't used it much (granted I'm probably mid-game right now)

[Brandon's Core](https://www.curseforge.com/minecraft/mc-mods/brandons-core): Draconic Evolution dependency

[ProjectE](https://www.curseforge.com/minecraft/mc-mods/projecte): This makes life a lot easier since you can dump all your extra cobble and such and get something useful back. I don't think it is too overpowered since the EMC values are at least a little balanced. This has saved me so much time by not having to craft stuff, with the exception of EnderIO items that use the "grains of infinity", which have no EMC.

**Comment by Lumpous** [comment]
Can't speak for other players.
ProjectE ruing gameplay pretty quick as there is no need anymore to actually build contrapions to get certain materials for other projects.
Tho i agree that having very limited materials have an emc-value to recycle and reuse that emc can be quite handy.
Like OP mentioned - getting rid of cobble and get something in exchange for it.

On the other hand: once a basic ME-Network is established, no one complains on 60k cobblestone in the network.
Even better, it can be used to be auto-processed down to gravel and once more to sand and auto smelted into glas.
Like with many other stuff that the ME-Network can handle.

So ProjectE: Maybe, with very limited blocks that have emc-values.

**Comment by james-pre** [comment]
Yeah, those are some good points. Perhaps we can do the other mods now and leave ProjectE up to the user?

**Comment by Lykayos** [comment]
ProjectE ruins gameplay

Metallurgy adds ways of ignoring resistance and armor on targets for easy 1 shots (bad for gameplay)
- Also adds weapons that scale with strength and have a base attack speed of 20 

Draconic Evolution adds more than endgame gear, it adds things that make the player unkillable, invulnerable to nearly everything, and infinite energy generation and storage. Weapons to destroy anything your heart could desire.

These are all extreme power creep mods and really not in line with the philosophy for this pack. 
These would all find a much better home in the "All the Mods" line of packs.
